lightning/src/onion_message/messenger.rs

```diff
@@ -262,6 +263,7 @@ pub struct OnionMessenger<
 	intercept_messages_for_offline_peers: bool,
 	pending_intercepted_msgs_events: Mutex<Vec<Event>>,
 	pending_peer_connected_events: Mutex<Vec<Event>>,
+	pending_events_processor: AtomicBool,
```

Comment:
Not sure I understand why "processor" is included in the naming. What is the bool suppose to signify?